runtime/gc_modron_startup/mminit.cpp

```diff
@@ -355,7 +355,7 @@ j9gc_initialize_heap(J9JavaVM *vm, IDATA *memoryParameterTable, UDATA heapBytesR
 
 			loadInfo->fatalErrorStr = (char *)j9mem_allocate_memory(formatLength, OMRMEM_CATEGORY_MM);
 			if (loadInfo->fatalErrorStr) {
-				j9str_printf(PORTLIB, loadInfo->fatalErrorStr, formatLength, format, size, qualifier);
+				j9str_printf(PORTLIB, (char *)loadInfo->fatalErrorStr, formatLength, format, size, qualifier);
```

Comment:
Please add a new local variable to receive the allocation which can be used here instead of the cast.
```c
			char *buffer = (char *)j9mem_allocate_memory(formatLength, OMRMEM_CATEGORY_MM);
			loadInfo->fatalErrorStr = buffer;
			if (NULL != buffer) {
				j9str_printf(PORTLIB, buffer, formatLength, format, size, qualifier);
				loadInfo->loadFlags |= FREE_ERROR_STRING; /* indicates that buffer should be freed later */
			}
```
The pattern is repeated several times.